packages/react-devtools-timeline/src/CanvasPage.js

```diff
@@ -170,6 +171,32 @@ function AutoSizedCanvas({
     [],
   );
 
+  const {searchIndex, searchRegExp, searchResults} = useContext(
+    TimelineSearchContext,
+  );
+
+  useLayoutEffect(() => {
```

Comment:
might be useful to add a short description of the intended behavior of this effect